.circleci/organise_doc.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const fs = require('fs-extra');
+const args = require('minimist')(process.argv.slice(2));
```

Comment:
`process.argv.slice(2)` will return an empty array if the argument isn't specified. Is this what you want it to do? If not, you should check the arguments above this line and error with a usage message.